subsys/partition_manager/flash_map_partition_manager.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#define DT_DRV_COMPAT fixed_partitions
+
+#include <zephyr.h>
+#include <storage/flash_map.h>
+
+/**
+ * Have flash_map_default use Partition Manager information instead of
+ * DeviceTree information when we are using the Partition Manager.
```

Comment:
Could we remove this comment now that this is in its own partition manager specific file?

Reply:
Yes.